python/ray/data/_internal/block_batching/interfaces.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import Any
+
+from ray.types import ObjectRef
+from ray.data.block import Block
+
+
+@dataclass
+class Batch:
+    """A batch of data.
```

Comment:
Say more? Data ready to return to the user post collation?

Reply:
added some more